lab-brianna/route/note-routes.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const Note = require('../model/note');
+
+module.exports = function(router) {
+  router.get('/api/note', (req, res) => {
```

Comment:
**TODO FOR LOST POINTS**

Get route doesn't work. Also, I know you're just starting off, but as noted during one of our sessions, you'll be abstracting your Mongoose methods into a separate controller module in the lib folder.